### PR TITLE
Adding an AeroGauge preset

### PR DESCRIPTION
**Pull request by LLONSIT**

### CODE REVIEW

File: backend/coreapp/compilers.py [hunk 1]
```diff
@@ -854,6 +854,8 @@ def preset_from_name(name: str) -> Optional[Preset]:
         "-O2 -g3 -mips2 -woff 624",
         diff_flags=["-Mreg-names=32"],
     ),
+    
+    Preset("AeroGauge", IDO53, "-O2 -g0 -mips2 -32 "),
```
Comment (EllipticEllipsis) [inline backend/coreapp/compilers.py:858]:
```suggestion
    Preset("AeroGauge", IDO53, "-O2 -mips2"),
```
`-32` is passed by `IDO53` already, `-g0` is the default so it's fine to leave off.

**Review by EllipticEllipsis — APPROVED** [review]

**Comment by EllipticEllipsis** [comment]
Could you run `black` on the branch? <https://github.com/decompme/decomp.me/blob/main/docs/CONTRIBUTING.md#linting>

**Comment by ethteck** [comment]
I think he's editing directly on the site. The issue is the empty line before the new preset - try removing that

**Comment by LLONSIT** [comment]
> I think he's editing directly on the site. The issue is the empty line before the new preset - try removing that

Done

**Pull request merged**